index.d.ts

```diff
@@ -113,31 +113,84 @@ declare namespace slugify {
 	}
 }
 
-/**
-Slugify a string.
+declare const slugify: {
+	/**
+	Slugify a string.
 
-@param string - String to slugify.
+	@param string - String to slugify.
 
-@example
-```
-import slugify = require('@sindresorhus/slugify');
+	@example
+	```
+	import slugify = require('@sindresorhus/slugify');
 
-slugify('I ♥ Dogs');
-//=> 'i-love-dogs'
+	slugify('I ♥ Dogs');
+	//=> 'i-love-dogs'
 
-slugify('  Déjà Vu!  ');
-//=> 'deja-vu'
+	slugify('  Déjà Vu!  ');
+	//=> 'deja-vu'
 
-slugify('fooBar 123 $#%');
-//=> 'foo-bar-123'
+	slugify('fooBar 123 $#%');
+	//=> 'foo-bar-123'
 
-slugify('я люблю единорогов');
-//=> 'ya-lyublyu-edinorogov'
-```
-*/
-declare function slugify(
-	string: string,
-	options?: slugify.Options
-): string;
+	slugify('я люблю единорогов');
+	//=> 'ya-lyublyu-edinorogov'
+	```
+	*/
+	(
+		string: string,
+		options?: slugify.Options
+	): string;
+
+	/**
+	Creates a counter to ensure uniqueness.
```

Comment:
This and the readme description should be in sync. The description is different in the readme. Also missing the `#### Use case example of counter` part.